sdcm/provision/scylla_yaml/cluster_builder.py

```diff
@@ -0,0 +1,198 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2021 ScyllaDB
+from functools import cached_property
+from typing import Optional
+
+import attr
+
+from sdcm.provision.common.builders import builder_property, ValidationError, AttributeProvider
+from sdcm.provision.scylla_yaml.auxiliaries import ScyllaYamlBuilderBase
+
+
+@attr.attrs(kw_only=True, order=False, str=False, init=True)
+class ScyllaYamlClusterBuilder(ScyllaYamlBuilderBase):
+    openldap_server_address_port: str = attr.attrib(default=None, metadata={'input_param': True}, eq=False)
+    msldap_server_info: dict = attr.attrib(default=None, metadata={'input_param': True}, eq=False)
+
+    @property
+    def hinted_handoff_enabled(self) -> Optional[str]:
+        if self.params.get('hinted_handoff') == 'enabled':
+            return 'enabled'
+        if self.params.get('hinted_handoff') == 'disabled':
+            return 'disabled'
+        return None
+
+    @property
+    def experimental(self) -> Optional[bool]:
+        if self.params.get('experimental') is None:
+            return None
+        return bool(self.params.get('experimental'))
```

Comment:
This is wrong translation to boolean.
Because ```bool("False")``` returns ```True```.

It must be something like following:

```
return self.params.get('experimental').lower() in ("1", "true", "yes", "y")
```

Reply:
https://github.com/scylladb/scylla-cluster-tests/blob/0f4cb956b92e1443e08cdc7879114810e2023375/sdcm/sct_config.py#L77-83